examples/scaffolding/contrib/mcp/mcptest.py

```
@@ -28,7 +28,7 @@ def parse_arguments():
 from openai import AsyncOpenAI
```

Ruff (0.14.1)

Redefinition of unused `AsyncOpenAI` from line 4

Remove definition: `AsyncOpenAI`

(F811)